### PR TITLE
AI Workflows and Notifications changes

### PR DESCRIPTION
**Pull request by Dani-Relic**
# Description

Small changes and corrections to the docs of the Workflow, Channel, Destination resources. 

- [ ] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] My code is formatted to [Go standards](https://go.dev/blog/gofmt)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes. Go [here](https://github.com/newrelic/terraform-provider-newrelic#testing) for instructions on running tests locally.

## How to test this change?

Please describe how to test your changes. Include any relevant steps in the UI, HCL file(s), commands, etc

- Step 1
- Step 2
- etc

### CODE REVIEW

**Review by shiva-bandari — APPROVED** [review]

File: website/docs/r/notification_channel.html.markdown [hunk 1]
```diff
@@ -74,7 +75,7 @@ Each notification channel type supports a specific set of arguments for the `pro
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The ID of the channel.
-
+* `status` - The status of the channel can indicate a configuration issue. The default status is DEFUALT.
```
Comment (lzaga-newrelic) [inline website/docs/r/notification_channel.html.markdown:78]: This is not an outcome when you create a channel, only the id is an outcome:

Screen Shot 2022-09-05 at 12 21 55
---
The status is a computed attribute in the schema and you're seeing it in the plan:

Screen Shot 2022-09-05 at 12 24 42

File: website/docs/r/notification_destination.html.markdown [hunk 2]
```diff
@@ -82,6 +82,7 @@ Each notification destination type supports a specific set of arguments for the
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The ID of the destination.
+* `status` - The status of the destination can indicate a configuration issue. The default status is DEFUALT.
```
Comment (lzaga-newrelic) [inline website/docs/r/notification_destination.html.markdown:85]: same here

**Review by lzaga-newrelic — CHANGES_REQUESTED** [review]

File: website/docs/r/notification_channel.html.markdown [hunk 3]
```diff
@@ -36,7 +37,7 @@ The following arguments are supported:
 
 * `account_id` - (Optional) Determines the New Relic account where the notification channel will be created. Defaults to the account associated with the API key used.
 * `name` - (Required) The name of the channel.
-* `type` - (Required) The type of channel.  One of: `EMAIL`, `SERVICENOW_INCIDENTS`, `WEBHOOK`, `JIRA_CLASSIC`, `JIRA_NEXTGEN`, `PAGERDUTY_ACCOUNT_INTEGRATION` or `PAGERDUTY_SERVICE_INTEGRATION`.
```
Comment (lzaga-newrelic) [inline website/docs/r/notification_channel.html.markdown:39]: This also needs to be removed from the available types in the resource file

**Comment by lzaga-newrelic** [comment]
This PR's changes added to [this](https://github.com/newrelic/terraform-provider-newrelic/pull/1990) PR.

**Pull request closed without merging**